src/Action/Lti/GetUserAssignmentLtiLinkAction.php

```diff
@@ -88,7 +88,7 @@ public function __invoke(UserInterface $user, int $assignmentId): Response
             return $this->responder->createJsonResponse($ltiRequest);
         } catch (AssignmentNotFoundException $exception) {
             throw new NotFoundHttpException($exception->getMessage());
```

Comment:
we could also add previous exception here :)

Reply:
Fixed